### PR TITLE
Brig physician reversion thing which means that now they are a member of sec, but dont get sec gear but do get proper access to their shiz

### PR DESCRIPTION
**Pull request by PowerfulBacon**
## About The Pull Request

Makes brig physician a member of security.
Brig physician now spawns with a mindshield.
Brig physician access is now an actual access titled 'brig infirmary'
(From pre-change) brig physician no longer has security or brig access, meaning they cannot get access to security gear.

Additionally, while I am here, this PR removes keycard auth from security personnel, because there is absolutely no reason non-command should have keycard auth (Security is just as good as regular crew and shouldn't be given command tools for being sec).

## Why It's Good For The Game

Unlike the lawyer, a brig physician actually works for sec rather than independently from security so it doesn't work as a civilian position.
A lot of people don't like the changes that happened with the brig phys update [citation needed], which was made to give less security gear to them however was way too extreme.
This PR brings back the brig physician everyone knows and loves while removing their access to security tools at the same time.
Since the brig physician isn't expected to be in medical, they answer to the HOS not the CMO.
Affects brig physician and sec players pretty much only, and so far every single person that plays sec or brig physician has said negative things about this change.

If brig physicians are going to be consistent and in line with departmental sec (acting as a departmental medical) then they should be both a member of medical and their respective department (security), much like how departmental sec are a member of security and their respective department they are assigned to.

## Changelog
:cl:
add: Brig physician is now mindshielded.
add: Brig physician access now works properly.
add: Uses brig phys access on some maps to allow the brig phys to reach the surgery rooms.
del: Removes security officer's keycard auth access.
tweak: Brig physician now answers to the HOS instead of the CMO.
/:cl:

### CODE REVIEW

**Comment by PowerfulBacon** [comment]
obviously being a coder I didnt even check if it launches

File: code/modules/clothing/suits/jobs.dm [hunk 1]
```diff
@@ -74,7 +74,7 @@
 	item_state = "detsuit"
 
 //Brig Phys
-/obj/item/clothing/suit/hazardvest/brig_phys
+r
```
Comment (PowerfulBacon) [inline code/modules/clothing/suits/jobs.dm]:
```suggestion
/obj/item/clothing/suit/hazardvest/brig_phys
```

**Comment by Naevii** [comment]
speedmerge

**Comment by PowerfulBacon** [comment]
actually im testing it now

File: code/modules/jobs/job_types/brigphys.dm [hunk 2]
```diff
@@ -16,10 +16,11 @@
 
 	outfit = /datum/outfit/job/brig_phys
 
-	access = list(ACCESS_SEC_DOORS, ACCESS_COURT, ACCESS_MAINT_TUNNELS, ACCESS_MORGUE, ACCESS_MEDICAL)
-	minimal_access = list(ACCESS_SEC_DOORS, ACCESS_COURT, ACCESS_MAINT_TUNNELS, ACCESS_MORGUE, ACCESS_MEDICAL)
+	access = list(ACCESS_BRIGPHYS, ACCESS_SEC_DOORS, ACCESS_COURT, ACCESS_MAINT_TUNNELS, ACCESS_MORGUE, ACCESS_WEAPONS, ACCESS_MEDICAL, ACCESS_KEYCARD_AUTH)
+	minimal_access = list(ACCESS_BRIGPHYS, ACCESS_SEC_DOORS, ACCESS_COURT, ACCESS_MAINT_TUNNELS, ACCESS_MORGUE, ACCESS_WEAPONS, ACCESS_MEDICAL, ACCESS_KEYCARD_AUTH)
```
Comment (PowerfulBacon) [inline code/modules/jobs/job_types/brigphys.dm]: WHY DID THEY HAVE ACCESS_KEYCARD_AUTH!

**Comment by PowerfulBacon** [comment]
Ok the brig phys access actually works and is assignable on the HOP's console.

**Comment by VictorVow** [comment]
Originally Sec was given keycard auth just for changing alert level, since it doesn't make sense that it's dependent on say getting the CMO to agree that nuke ops are a red alert threat and getting to their office to auth. Instead the Warden and HoS can jointly red alert, or other Security in the event of HoS/Warden death and a red alert is still urgently required. 

Wasn't the best implementation though since keycard auth has access to other things like BSA, which isn't required here. It should probably be split into two different access levels.

**Comment by PowerfulBacon** [comment]
Normal seccies shouldnt be able to change the alert at all. Its not their job, it shouldn't be. Just because they have somewhat of a metashield doesn't mean they should be given more than it makes sense for them to have.

**Comment by VictorVow** [comment]
In the event of HoS being killed, then Warden and another seccie should be able to imo. It's almost always a dire situation if the HoS is murdered or missing.

**Comment by PowerfulBacon** [comment]
they cant even get into the places with keycard auth anyway. Since its in the name of security they should only have as much access as they need and nothing more, otherwise that's a security flaw.

**Comment by yyzsong** [comment]
Should probably keep CMO as a head. They *are* still a doctor after all.

**Comment by Naevii** [comment]
> Should probably keep CMO as a head. They _are_ still a doctor after all.

they are more dependant on security  rather than on medical, they spend most of their shift in security and hardly ever interact with medbay.

**Comment by VictorVow** [comment]
> Since its in the name of security they should only have as much access as they need and nothing more, otherwise that's a security flaw.

Agreed, and what I'm suggesting here is that setting alert level makes more sense as a Security function primarily and Command secondary. After all it's focused on "what threats is the station facing", how useful is the RD's input here even? HoS' responsibility should be keeping the crew updated on how alert to threats they should be.

This is separate from the discussion about keycard auth, which I agree isn't the ideal implementation at the moment. Ideally Sec would only have the ability to set red alert if needed and not the rest of keycard functions.

**Comment by PowerfulBacon** [comment]
eh sec should have no access to keycards or anything related to command duties, they are on the level of normal crewmember

**Comment by Rukofamicom** [comment]
This PR is one of the two ideal solutions to the brig physician problems. Brig physician is now primarily a doctor for Sec and inmates. Innate access to security equipment should never have been something they had because people shouldn't be playing brig physician with the intent to act/arm up as an officer - but the option remains available with HoS approval in times of need. 

> sec should have no access to keycards or anything related to command duties

I second this and it's a nice bonus to see this as a part of the PR. Security should not have near unilateral control over alert levels when alert levels are the primary limiting factor on what security is and isn't allowed to do. Security deserves to be held in check just like every other department and it should be up to command staff whether or not they get to kick into powergamer mode or not.

Security exists to enforce space law, not to speed-rush antags into an execution chamber by boosting themselves to blue/red alert because someone has a poppy in their hair.

File: code/modules/jobs/job_types/brigphys.dm [hunk 3]
```diff
@@ -1,12 +1,12 @@
 /datum/job/brig_phys
 	title = "Brig Physician"
 	flag = BRIG_PHYS
-	department_head = list("Chief Medical Officer")
+	department_head = list("Head of Security")
```
Comment (PowerfulBacon) [inline code/modules/jobs/job_types/brigphys.dm]:
```suggestion
	department_head = list("Head of Security", "Chief Medical Officer")
```

File: code/modules/jobs/job_types/brigphys.dm [hunk 4]
```diff
@@ -1,12 +1,12 @@
...
 	department_flag = ENGSEC
 	faction = "Station"
 	total_positions = 1
 	spawn_positions = 1
-	supervisors = "chief medical officer"
+	supervisors = "the head of security"
```
Comment (PowerfulBacon) [inline code/modules/jobs/job_types/brigphys.dm]:
```suggestion
	supervisors = "the head of security and the chief medical officer"
```

**Comment by VictorVow** [comment]
> Security deserves to be held in check just like every other department and it should be up to command staff whether or not they get to kick into powergamer mode or not.

This is already a powergaming rule violation so it seems redundant to me. If people really are abusing that by frivolously raising threat levels then just ban them for it.

**Review by Firespider01 — APPROVED** [review]

**Review by Crossedfall — CHANGES_REQUESTED** [review]
Alright. I've sat down and talked to Bacon about this for a bit and I've shifted from being entirely neutral to being for the original change.

The main thing for me is the inconsistencies presented in how the brig physician role is handled. It is not part of security. It was slowly changed to be that against its original design and forced into being part of security, but, looking at it objectively from a design point of view that really doesn't make any sense.

A brig physician is almost identical in function to a department security officer. In-universe, department officers are expected to have a basic understanding of the department they are stationed in so that they can lend a hand if absolutely necessary; their primary focus though is to keep that area safe. They have an office in that department, have access to their radio channel, and even has some basic access to the area itself. **BUT**, they are not part of that department. They are part of security.

Similarly... Brig physicians have an office in security, help out with medical related issues, and have access to their radio channel, but they are not part of security. They are a medical officer tasked with servicing that area. *Just like how a department security officer is tasked with serving their assigned area*.

In terms of design, I see no reason why they would be treated as an exception to this long established rule of interdepartmental co-existence.

Going to have to gather more opinions on this, but for now I'm going to slap the DNM on this as it's currently written.

**Comment by Naevii** [comment]
> Alright. I've sat down and talked to Bacon about this for a bit and I've shifted from being entirely neutral to being for the original change.
> 
> The main thing for me is the inconsistencies presented in how the brig physician role is handled. It is not part of security. It was slowly changed to be that against its original design and forced into being part of security, but, looking at it objectively from a design point of view that really doesn't make any sense.
> 
> A brig physician is almost identical in function to a department security officer. In-universe, department officers are expected to have a basic understanding of the department they are stationed in so that they can lend a hand if absolutely necessary; their primary focus though is to keep that area safe. They have an office in that department, have access to their radio channel, and even has some basic access to the area itself. **BUT**, they are not part of that department. They are part of security.
> 
> Similarly... Brig physicians have an office in security, help out with medical related issues, and have access to their radio channel, but they are not part of security. They are a medical officer tasked with servicing that area. _Just like how a department security officer is tasked with serving their assigned area_.
> 
> In terms of design, I see no reason why they would be treated as an exception to this long established rule of interdepartmental co-existence.
> 
> Going to have to gather more opinions on this, but for now I'm going to slap the DNM on this as it's currently written.

because players who played this role, and security liked this role the way it was BEFORE the "good change"?

if brig physician can't be mindshielded it just allows shitters for easier antag role with more access.

Similarly, there would be no reason to have a person, stationing in brig if you can't even trust them and they exist to make your work harder. 
And as a brig physician one trick pony i can say i would rather see this job entirely removed rather than make it into second lawyer.

**Comment by Crossedfall** [comment]
Discussions moved to discord, since having prolonged debate is difficult on github.

**Comment by PowerfulBacon** [comment]
So is on discord cause the messages get lost after 5 minutes (and I have no idea where the discussion even is)

**Comment by Crossedfall** [comment]
> 
> 
> So is on discord cause the messages get lost after 5 minutes (and I have no idea where the discussion even is)

It's on discord because github isn't a good place for prolonged discussion and never has been. I don't like Discord as a main alternative, and would prefer the forums, but this is how it went down.

**Comment by utilbird** [comment]
better emotes on discord

**Comment by PowerfulBacon** [comment]
I am almost perfectly split at this point between making them a member of sec and not since the arguments for both sides are pretty strong.

To me it would be like departmental sec, as like you said, departmental sec are expected to have basic understanding of the department, answer to both heads and are a member of both security and their respective department at the same time. Brig physician should be both am member of security and medical at the same time if we follow that, original version of this PRw as flawed and did not have them answering to the CMO however that has been changed.

**Comment by Sapfearon** [comment]
Brig phys worked like IRL police medic - he is medic, he is member of police AND has relevant police training. Yes, he is not carrying guns nor arrest, but he is trusted member of police and not some random medic assigned to area.
Which is why reverting him back to being mindshielded is good idea.
By the way, why detective is job rank lower then sec officer? He is mindshield, he is not private detective and irl detective is big step up from normal grunt. Should be same way here imo. Opens up slot for non-mindshielded private investigator too.

**Comment by Socialistion** [comment]
Add this please brig phys is one of my favorite jobs and I quite enjoyed being a sec doctor. I don't have much to actually contribute to any discussion but please, removing brig phys mindshield was probably the most idiotic thing that could've been done in my opinion.

**Comment by PowerfulBacon** [comment]
@Crossedfall yay or nay

**Pull request closed without merging**